### PR TITLE
ittms staging

### PR DESCRIPTION
**Pull request by temitope777**
## Trello
https://trello.com/c/IBi5LAS5/921-ittms-create-staging-environment

## Context

To create the staging environment for ITTMS

## Changes proposed in this pull request

Added the staging target to make file and the relevant env, rb and tfvars files

## Guidance to review

make staging terraform-plan
## Things to check

- [ ] If the code removes any existing feature flags, a data migration has also been added to delete the entry from the database
- [ ] This code does not rely on migrations in the same Pull Request
- [ ] If this code includes a migration adding or changing columns, it also backfills existing records for consistency
- [ ] If this code adds a column to the DB, decide whether it needs to be in analytics yml file or analytics blocklist
- [ ] API release notes have been updated if necessary
- [ ] If it adds a significant user-facing change, is it documented in the [CHANGELOG](CHANGELOG.md)?
- [ ] Required environment variables have been updated or [added to the Azure KeyVault](/docs/environment-variables.md#deploy-pipeline)

### CODE REVIEW

File: config/environments/staging.rb
Comment (Nitemaeric) [inline config/environments/staging.rb:1]: We previously decided to go against adding `RAILS_ENV` variants. All deployed apps will use `RAILS_ENV=production`.

That means, this file and the `.env.staging` are not needed.

Comment (temitope777) [reply config/environments/staging.rb:1]: removed

File: terraform/application/config/stg_Terrafile [hunk 1]
```diff
@@ -0,0 +1,3 @@
+aks:
+    source:  "https://github.com/DFE-Digital/terraform-modules"
+    version: "main"
```
Comment (saliceti) [inline terraform/application/config/stg_Terrafile:3]: It should be "testing"

File: Makefile [hunk 2]
```diff
@@ -30,6 +30,14 @@ qa: test-cluster
 		$(eval export TF_VAR_environment=${ENVIRONMENT})
 		$(eval include global_config/qa.sh)
 
+
+.PHONY: staging
+staging: test-cluster
+		$(eval ENVIRONMENT=staging)
+		$(eval export TF_VAR_environment=${ENVIRONMENT})
+		$(eval include global_config/staging.sh)
+
```
Comment (saliceti) [inline Makefile:39]: Please remove extra blank line

File: global_config/staging.sh [hunk 3]
```diff
@@ -0,0 +1,5 @@
+CONFIG=stg
+CONFIG_SHORT=stg
+AZURE_SUBSCRIPTION=s189-teacher-services-cloud-test
+AZURE_RESOURCE_PREFIX=s189t01
+KV_PURGE_PROTECTION=false
```
Comment (saliceti) [inline global_config/staging.sh:5]: Please add missing end of file lines

File: Makefile (hunk 2 above)
Comment (temitope777) [reply Makefile:39]: done

File: global_config/staging.sh (hunk 3 above)
Comment (temitope777) [reply global_config/staging.sh:5]: added

**Comment by temitope777** [comment]
[ITTMS: Create staging environment](https://trello.com/c/IBi5LAS5/921-ittms-create-staging-environment)

**Pull request merged**